app/src/main/java/com/appunite/loudius/ui/reviewers/ReviewersViewModel.kt

```diff
@@ -74,7 +74,7 @@ class ReviewersViewModel @Inject constructor(
         fetchData()
     }
 
-    private fun fetchData() {
+    fun fetchData() {
```

Comment:
The same as above

Reply:
No, it's not. When user doesn't pull to refresh, the old data is still presented.
I don't think this test is necessary because the `WHEN init THEN display pull requests list` test is doing this job.